### ISSUE TITLE
Add new entity to the YAML model for "stand alone" attributes

### ISSUE THREAD

**Issue by joaopgrassi** [open-telemetry/build-tools#202]
The semantic conventions SIG has agreed to go forward with the proposal https://github.com/open-telemetry/semantic-conventions/issues/197 to decouple the attribute definition from its usage in the conventions. Essentially, we will achieve a "registry" of attributes that are standalone and then can be referenced by any convention. 

During the initial split for HTTP https://github.com/open-telemetry/semantic-conventions/pull/208 we saw that the current YAML model is not fit for this. The attributes entity today have many properties such as requirement level that are not relevant for the registry. For ex, the requirement level is rendered in markdown, which does not apply for the registry concept. 

As a temporary workaround, @AlexanderWert worked on #190 to unblock working on the registry "move". But in the long run, we should have a better solution, that involves modifying our YAML model and introduce a new entity for such attribute registry type. 

@AlexanderWert  also worked on a proposal for this, which can be seen in #191

**Comment by Oberon00** [comment]
IMHO, attribute_group could (should) be adapted to fulfill these instead of adding yet another type

**Comment by lmolkova** [comment]
Related to #290

**Comment by jsuereth** [comment]
Closing as wontfix.  We've migrated attribute registry to weaver with policy enforcement.  I think there's an overall question of how we evolve our YAML model to better account for a registry and formalize things in group, e.g.
